.github/workflows/v2-merge-forward.yml

```diff
@@ -0,0 +1,29 @@
+name: 'v2-main merge forward'
+on:
+  schedule:
+    - cron: '0 9 * * 1,3,5' # 9AM UTC every M/W/F
+
+jobs:
+  merge:
+    name: Merge
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout & Merge
+        uses: actions/checkout@v2
+        with:
+          ref: 'v2-main'
+        run: |
+          git merge master
+      - name: Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: v2/forward-merge
```

Comment:
Where is the part that creates a branch from v2 to v2/forward-merge?

Reply:
As far as I can tell, here's how this works -

The `ref` at line 14 checks out the `v2-main` branch locally and later runs `git merge master` (line 16).

The `create-pull-request` GH action, takes care of pushing these new changes to a branch named `v2/forward-merge` and creates a pull request against the specified `base` branch.

From the [documentation](https://github.com/peter-evans/create-pull-request#-create-pull-request) -

>Create Pull Request action will:
>    1. Check for repository changes in the Actions workspace. This includes:
>        * untracked (new) files
>        * tracked (modified) files
>        * commits made during the workflow that have not been pushed
>    2. Commit all changes to a new branch, or update an existing pull request branch.
>    3. Create a pull request to merge the new branch into the base—the branch checked out in the workflow.

Reply:
Okay. Looks good